docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -6,6 +6,16 @@ The MySQL connector allows querying and creating tables in an external
 `MySQL <https://www.mysql.com/>`_ instance. This can be used to join data between different
 systems like MySQL and Hive, or between two different MySQL instances.
 
+Requirements
+------------
+
+Requirements for using the connector in a catalog to connect to a MySQL data
+source are:
+
+* MySQL 5.6, 5.7, 8.0 and higher
```

Comment:
Notably not everything works with MySQL < 8:

- Unicode values don't work properly out of box
- RENAME COLUMN isn't supported

Reply:
The language here is also different from others: `MySQL 5.6 and higher`?

Reply:
yeah ... thats just the text from the JDBC driver ... we can change it to whatever makes most sense. E.g. is there a MySQL 6 or 7 that is not supported?

Reply:
There was no 6 or 7. :trollface: 

It went 5.6, 5.7 and 8.0.